src/test/java/com/fasterxml/jackson/databind/MapperFootprintTest.java

```diff
@@ -0,0 +1,18 @@
+package com.fasterxml.jackson.databind;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openjdk.jol.info.GraphLayout;
+
+public class MapperFootprintTest {
+    @Test
+    @Ignore
+    public void testMapperFootprint() {
+        // memory footprint limit for the ObjectMapper
+        GraphLayout mapperLayout = GraphLayout.parseInstance(new ObjectMapper())
+                .subtract(GraphLayout.parseInstance(new ObjectMapper()));
+        System.out.println(mapperLayout.toFootprint());
```

Comment:
I assume it was left during debugging; either remove (preferably I think) or add a prefix statement explaining what and why is being printed.

Reply:
done, I've made it only print the full footprint if the assertion fails, now